kafka-connect-s3/src/main/java/io/confluent/connect/s3/storage/S3OutputStream.java

```diff
@@ -126,13 +124,8 @@ private void uploadPart(final int size) throws IOException {
       multiPartUpload = newMultipartUpload();
     }
     try {
-      retry(new Runnable() {
-        public void run() {
-          multiPartUpload.uploadPart(new ByteArrayInputStream(buffer.array()), size);
-        }
-      }, retries, "Part upload failed");
```

Comment:
After removing this code, `retries` member variable is unused and needs to be removed too.

Reply:
Good catch, thanks, updated.